include/oneapi/dpl/pstl/algorithm_impl.h

```diff
@@ -1108,8 +1107,8 @@ struct __brick_copy<_Tag, _ExecutionPolicy, ::std::enable_if_t<__is_host_dispatc
 // move
 //------------------------------------------------------------------------
 
-template <class _Tag, typename _ExecutionPolicy>
-struct __brick_move<_Tag, _ExecutionPolicy, ::std::enable_if_t<__is_host_dispatch_tag_v<_Tag>>>
+template <class _Tag>
+struct __brick_move<_Tag, std::enable_if_t<__is_host_dispatch_tag_v<_Tag>>>
```

Comment:
The same comment as above.
(So, we don't need std::enable_if_t here.)

Reply:
Fixed